hardware/Schematic Design Guide.md

```diff
@@ -0,0 +1,148 @@
+# Schematic Design Guide
+The customer want to design its own schematic of project. First of all the customer should find a reference from Silabs' web source. Then study the schematic with hardware design consideration and RF circuits matching guide if interest. And make its own schematic based on the reference design and actual applications.
+
+  * [Find the relative reference design from Silabs' website](#find-the-relative-reference-design-from-silabs-website)
+  * [Hardware design considerations](#hardware-design-considerations)
+  * [RF circuits matching guide](#rf-circuits-matching-guide)
+  * [Customize the schematic design to fit the application](#customize-the-schematic-design-to-fit-the-application)
+## Find the relative reference design from Silabs' website
+There are relative reference designs for all of the solutions in the website. Basically the reference designs differ from frequency bands, output power levels, DC-DC regulator configurations, and chip packages. 
+- Frequency bands: 
+
+  2.4GHz for: Proprietary, BLE, and Zigbee;
+
+  Sub-G for:  Proprietary, Z-Wave.
+
+- Output power levels: 
+
+  0dBm, +8dBm, +10dBm, +13~14dBm, +20dBm.
+- DC-DC regulator configuration: 
+
+  Use it for power efficiency; 
+ 
+  Don’t use it for saving cost.
+
+- Chip packages:
+
+  32/48/68 pins QFN package and 125pins BGA package for EFR32 chips.
+
+For example, you want to search the reference schematic of proprietary chip, you can go to the following web page:
+
+![](../files/HW-Schematic-Design-Guide/Proprietary-Page.png)
+In this page, you can move mouse to Products tab, then the pop-down menu will appear with Wireless products which include the Proprietary item, then click left button of your mouse. And it will enter the Proprietary product list page as following:
+![](../files/HW-Schematic-Design-Guide/Proprietary-Product.png)
+You can find the generations of proprietary products as listed in the bottom table. You can select EFR32FG1 series 1 sub-GHz and 2.4GHz SoCs for more details:
+![](../files/HW-Schematic-Design-Guide/Proprietary-S1.png)
+In this page, you can find the brief introduction of the proprietary series 1 chips and the parameter list of each chips. Ignore the chips list, you can click the View Document in this page and acess the documents list pages directly:
+![](../files/HW-Schematic-Design-Guide/Proprietary-Reference.png)
+For example, If you want to look for 434MHz +10dBm output power reference design, you can click item EFR32FG1 BRD4251B 2400/433 MHz 10dBm Radio Board Full Design Package and download the reference design package to your local address.
+The schematic likes:[BRD4251B](https://www.silabs.com/documents/public/schematic-files/EFR32FG1-BRD4251B-B00-schematic.pdf)
```

Comment:
Adding a space behind the colon makes it look better.

Reply:
Done.